quadriga/src/test/java/edu/asu/diging/quadriga/core/service/impl/CollectionManagerImplTest.java

```diff
@@ -132,6 +140,41 @@ public boolean matches(Collection arg0) {
         managerToTest.deleteCollection(id.toString());
     }
     
+    @Test
+    public void test_deleteCollection_archived() throws CollectionNotFoundException, InvalidObjectIdException {
+        String name = "name";
+        String desc = "description";
+        Collection collection = new Collection();
+        ObjectId id = new ObjectId();
+        collection.setId(id);
+        collection.setName(name);
+        collection.setDescription(desc);
+        
+        Collection response = new Collection();
+        collection.setId(id);
+        collection.setName(name);
+        collection.setDescription(desc);
+        collection.setArchived(true);
+        
+        Mockito.when(collectionRepo.findById(id)).thenReturn(Optional.of(collection));
+        Mockito.when(eventGraphService.findLatestEventGraphByCollectionId(id)).thenReturn(new EventGraph());
+        Mockito.when(collectionRepo.save(Mockito.any())).thenReturn(response);
+        
+        try{
+            managerToTest.deleteCollection(id.toString());
+        }
+        catch (CollectionNotFoundException e)
+        {
+            fail("Exception thrown during test execution: " + e.getMessage());
+            
+        }
+        catch(InvalidObjectIdException e)
+        {
+            fail("Exception thrown during test execution: " + e.getMessage());
+        }
```

Comment:
I don't think you need this, just let it throw the exceptions and the tests will fail automatically.

Reply:
when does the test pass though? There needs to be an assert statement.